weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/CellRenderers.tsx

```diff
@@ -0,0 +1,120 @@
+import {Box} from '@mui/material';
+import {
+  GridEditInputCell,
+  GridRenderCellParams,
+  GridRenderEditCellParams,
+} from '@mui/x-data-grid-pro';
+import {Icon} from '@wandb/weave/components/Icon';
+import React, {useState} from 'react';
+import styled from 'styled-components';
+
+const commonCellStyles = {
+  height: '100%',
+  width: '100%',
+  fontFamily: '"Source Sans Pro", sans-serif',
+  fontSize: '14px',
+  lineHeight: '1.5',
+  padding: '12px',
+  borderRadius: '2px',
+  transition: 'background-color 0.5s ease',
+};
+
+interface CellViewingRendererProps extends GridRenderCellParams {
+  isEdited?: boolean;
+  isDeleted?: boolean;
+  isNew?: boolean;
+}
+
+const StyledEditCell = styled(GridEditInputCell)`
+  textarea {
+    height: 100% !important;
+    padding: 12px;
+    padding-top: 26px;
+    font-family: 'Source Sans Pro', sans-serif;
+    vertical-align: top;
+  }
+  .MuiInputBase-root {
+    height: 100%;
+    align-items: flex-start;
+  }
+  .MuiInputBase-input {
+    height: 100% !important;
+  }
+`;
+
+export const CellViewingRenderer: React.FC<CellViewingRendererProps> = ({
+  value,
+  isEdited = false,
+  isDeleted = false,
+  isNew = false,
+  api,
+  id,
+  field,
+}) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const handleEditClick = (event: React.MouseEvent) => {
+    event.stopPropagation();
+    api.startCellEditMode({id, field});
+  };
+
+  const getBackgroundColor = () => {
+    if (isDeleted) {
+      return 'rgba(255, 0, 0, 0.1)';
+    }
+    if (isEdited) {
+      return 'rgba(0, 128, 128, 0.1)';
+    }
+    if (isNew) {
+      return 'rgba(0, 255, 0, 0.1)';
+    }
+    return 'transparent';
+  };
+
+  return (
+    <Box
```

Comment:
very small padding issue in the cell when it has been edited:

<img width="632" alt="Screenshot 2025-01-09 at 10 04 49 AM" src="https://github.com/user-attachments/assets/80b446d6-b2bb-4602-9b6b-c4ddec41b75f" />

Reply:
I fixed this, its a 1px pad coming from the line under the header row